File: C# Fundamentals/BasicSyntax/PadawanEquipment.cs
using System;

namespace PadawanEquipment
{
    class Program
    {
        static void Main(string[] args)
        {
            double budget = double.Parse(Console.ReadLine());
            int studentsCount = int.Parse(Console.ReadLine());
            double lightsaberPrice = double.Parse(Console.ReadLine());
            double robePrice = double.Parse(Console.ReadLine());
            double beltPrice = double.Parse(Console.ReadLine());

            double totalPrice = studentsCount * (robePrice + beltPrice) + Math.Ceiling(studentsCount * (1 + 10 / 100.0)) * lightsaberPrice;
            for (int i = 1; i <= studentsCount; i++)
            {
                if (i % 6 == 0)
                {
                    totalPrice -= beltPrice;
                }
            }

            if (totalPrice <= budget)
            {
                Console.WriteLine($"The money is enough - it would cost {totalPrice:f2}lv.");
            }
            else
            {
                Console.WriteLine($"Ivan Cho will need {totalPrice - budget:f2}lv more.");
            }
        }
    }
}

File: Java Fundamentals/BasicSyntaxEx/PadawanEquipment.java
package BasicSyntaxEx;

import java.util.Scanner;

public class PadawanEquipment {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        int studentsCount = Integer.parseInt(scanner.nextLine());
        double saberPrice = Double.parseDouble(scanner.nextLine());
        double robePrice = Double.parseDouble(scanner.nextLine());
        double beltPrice = Double.parseDouble(scanner.nextLine());

        double fundsNeeded = studentsCount * (robePrice) +
                Math.ceil(studentsCount * (1 + 10 / 100d)) * saberPrice +
                studentsCount * beltPrice - Math.floor(studentsCount / 6d) * beltPrice;

        if (fundsNeeded <= budget) {
            System.out.printf("The money is enough - it would cost %.2flv.", fundsNeeded);
        } else {
            System.out.printf("Ivan Cho will need %.2flv more.", fundsNeeded - budget);
        }
    }
}
